modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/WindowsAPIUtils.java

```diff
@@ -199,4 +199,12 @@ public static OAuth2TokenValidationService getOAuth2TokenValidationService() {
         return oAuth2TokenValidationService;
     }
 
+    public static void startTenantFlow(String userName) {
```

Comment:
Is this function having a valid use case, only saw one particular usage of this and that was also calling this function just before a function return.